builds/e2e/Runner.ps1

```diff
@@ -0,0 +1,151 @@
+New-Module -ScriptBlock {
+
+    [Console]::OutputEncoding = New-Object -typename System.Text.ASCIIEncoding
+
+    <#
+     # Completes initialization of a Windows VM (version 1809 or later) that is
+     # installed behind an HTTPS proxy server for end-to-end testing of Azure
+     # IoT Edge. It installs/enables sshd so that the test agent can SSH into
+     # this VM and Linux VMs using the same commands. It also configures system-
+     # wide proxy settings (so that sshd and related components can be
+     # downloaded) and sets the default SSH shell to a newer version of
+     # PowerShell (so that the agent can make use of the NoProxy parameter
+     # on Invoke-WebRequest rather than reverting system-wide proxy settings).
+     #>
+
+    #requires -Version 5
+    #requires -RunAsAdministrator
+
+    Set-Variable PwshInstallUri -Option Constant -Value 'https://github.com/PowerShell/PowerShell/releases/download/v6.1.1/PowerShell-6.1.1-win-x64.zip'
+    Set-Variable ProxySettingsModule -Option Constant -Value 'https://raw.githubusercontent.com/PowerShell/NetworkingDsc/8a4ae47b835b931d816dbbd5244d59e8fb0ce36b/Modules/NetworkingDsc/DSCResources/MSFT_ProxySettings/MSFT_ProxySettings.psm1'
```

Comment:
Hadn't thought of doing that...great idea. I wanted to avoid the overhead of using DSC in the deployment template, but there's no reason I can't just install the module (a much cleaner version of what I'm doing already).

Reply:
Done